### PR TITLE
python3Packages.tqdm: 4.58.0 -> 4.60.0

### PR DESCRIPTION
**Pull request by drewrisinger**
###### Motivation for this change

Noticed some things that could be updated when reviewing #119608.
Update to latest version: https://github.com/tqdm/tqdm/releases/tag/v4.60.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 119623` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>python38Packages.optuna</li>
<li>python38Packages.pymc3</li>
<li>python38Packages.shap</li>
<li>python38Packages.skorch</li>
<li>python38Packages.textacy</li>
<li>python39Packages.optuna</li>
<li>python39Packages.skorch</li>
<li>python39Packages.textacy</li>
</ul>
</details>
<details>

  <summary>3 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>apache-airflow: log was empty</li>
<li>python38Packages.pywick: log was empty</li>
<li>python39Packages.pywick: log was empty</li>
  </ul>
</details>
<details>

  <summary>2 packages failed to build and are new build failures:</summary>
  <ul>
<li>python38Packages.pytorch-metric-learning: log was empty</li>
<li>python39Packages.pytorch-metric-learning: log was empty</li>
  </ul>
</details>
<details>
<summary>
166 packages built:
</summary>
<ul>
<li>aiodnsbrute</li>
<li>backblaze-b2</li>
<li>check_systemd</li>
<li>checkov</li>
<li>conan</li>
<li>deja-dup</li>
<li>dpt-rp1-py</li>
<li>duplicity</li>
<li>duply</li>
<li>evdevremapkeys</li>
<li>evillimiter</li>
<li>ffmpeg-normalize</li>
<li>gdown (python38Packages.gdown)</li>
<li>gnome-keysign</li>
<li>home-assistant</li>
<li>jellyfin-mpv-shim</li>
<li>kaggle (python38Packages.kaggle)</li>
<li>keepkey_agent (python38Packages.keepkey_agent)</li>
<li>ldeep</li>
<li>ledger_agent (python38Packages.ledger_agent)</li>
<li>lieer</li>
<li>luigi</li>
<li>magic-wormhole (python38Packages.magic-wormhole)</li>
<li>manim</li>
<li>mathlibtools (python38Packages.mathlibtools)</li>
<li>mnemosyne</li>
<li>noto-fonts-emoji</li>
<li>nrfutil</li>
<li>ocrmypdf</li>
<li>papis (python38Packages.papis)</li>
<li>past-time</li>
<li>plex-mpv-shim</li>
<li>python38Packages.afdko</li>
<li>python38Packages.aiosmb</li>
<li>python38Packages.ansible-kernel</li>
<li>python38Packages.ansible-runner</li>
<li>python38Packages.b2sdk</li>
<li>python38Packages.baselines</li>
<li>python38Packages.cirq</li>
<li>python38Packages.datasets</li>
<li>python38Packages.datashader</li>
<li>python38Packages.dipy</li>
<li>python38Packages.dnachisel</li>
<li>python38Packages.genome-collector</li>
<li>python38Packages.gradient</li>
<li>python38Packages.gradient-utils</li>
<li>python38Packages.graphtage</li>
<li>python38Packages.gtts</li>
<li>python38Packages.holoviews</li>
<li>python38Packages.huggingface-hub</li>
<li>python38Packages.hvplot</li>
<li>python38Packages.hyperopt</li>
<li>python38Packages.ignite</li>
<li>python38Packages.intake</li>
<li>python38Packages.internetarchive</li>
<li>python38Packages.labelbox</li>
<li>python38Packages.libagent</li>
<li>python38Packages.lsassy</li>
<li>python38Packages.mesa</li>
<li>python38Packages.moviepy</li>
<li>python38Packages.msldap</li>
<li>python38Packages.nagiosplugin</li>
<li>python38Packages.nbsmoke</li>
<li>python38Packages.ndjson</li>
<li>python38Packages.nltk</li>
<li>python38Packages.nototools</li>
<li>python38Packages.panel</li>
<li>python38Packages.papermill</li>
<li>python38Packages.parfive</li>
<li>python38Packages.perfplot</li>
<li>python38Packages.plexapi</li>
<li>python38Packages.proglog</li>
<li>python38Packages.pyppeteer</li>
<li>python38Packages.pypykatz</li>
<li>python38Packages.pyro-ppl</li>
<li>python38Packages.python-daemon</li>
<li>python38Packages.python-miio</li>
<li>python38Packages.python-mpv-jsonipc</li>
<li>python38Packages.pytorch-lightning</li>
<li>python38Packages.rbtools</li>
<li>python38Packages.rising</li>
<li>python38Packages.sacremoses</li>
<li>python38Packages.salmon-mail</li>
<li>python38Packages.spacy</li>
<li>python38Packages.sunpy</li>
<li>python38Packages.thinc</li>
<li>python38Packages.toggl-cli</li>
<li>python38Packages.tqdm</li>
<li>python38Packages.transformers</li>
<li>trezor_agent (python38Packages.trezor_agent)</li>
<li>python38Packages.tubeup</li>
<li>python38Packages.twine</li>
<li>python39Packages.afdko</li>
<li>python39Packages.aiosmb</li>
<li>python39Packages.ansible-kernel</li>
<li>python39Packages.ansible-runner</li>
<li>python39Packages.b2sdk</li>
<li>python39Packages.baselines</li>
<li>python39Packages.cirq</li>
<li>python39Packages.datasets</li>
<li>python39Packages.dipy</li>
<li>python39Packages.dnachisel</li>
<li>python39Packages.gdown</li>
<li>python39Packages.genome-collector</li>
<li>python39Packages.gradient</li>
<li>python39Packages.gradient-utils</li>
<li>python39Packages.graphtage</li>
<li>python39Packages.gtts</li>
<li>python39Packages.holoviews</li>
<li>python39Packages.huggingface-hub</li>
<li>python39Packages.hvplot</li>
<li>python39Packages.hyperopt</li>
<li>python39Packages.ignite</li>
<li>python39Packages.intake</li>
<li>python39Packages.internetarchive</li>
<li>python39Packages.kaggle</li>
<li>python39Packages.keepkey_agent</li>
<li>python39Packages.labelbox</li>
<li>python39Packages.ledger_agent</li>
<li>python39Packages.libagent</li>
<li>python39Packages.lsassy</li>
<li>python39Packages.magic-wormhole</li>
<li>python39Packages.mathlibtools</li>
<li>python39Packages.mesa</li>
<li>python39Packages.moviepy</li>
<li>python39Packages.msldap</li>
<li>python39Packages.nagiosplugin</li>
<li>python39Packages.nbsmoke</li>
<li>python39Packages.ndjson</li>
<li>python39Packages.nltk</li>
<li>python39Packages.nototools</li>
<li>python39Packages.panel</li>
<li>python39Packages.papermill</li>
<li>python39Packages.papis</li>
<li>python39Packages.parfive</li>
<li>python39Packages.perfplot</li>
<li>python39Packages.plexapi</li>
<li>python39Packages.proglog</li>
<li>python39Packages.pyppeteer</li>
<li>python39Packages.pypykatz</li>
<li>python39Packages.pyro-ppl</li>
<li>python39Packages.python-daemon</li>
<li>python39Packages.python-miio</li>
<li>python39Packages.python-mpv-jsonipc</li>
<li>python39Packages.pytorch-lightning</li>
<li>python39Packages.rbtools</li>
<li>python39Packages.rising</li>
<li>python39Packages.sacremoses</li>
<li>python39Packages.salmon-mail</li>
<li>python39Packages.spacy</li>
<li>python39Packages.sunpy</li>
<li>python39Packages.thinc</li>
<li>python39Packages.tqdm</li>
<li>python39Packages.transformers</li>
<li>python39Packages.trezor_agent</li>
<li>python39Packages.tubeup</li>
<li>python39Packages.twine</li>
<li>r128gain</li>
<li>sacad</li>
<li>spotdl</li>
<li>tartube</li>
<li>theharvester</li>
<li>tinyprog</li>
<li>tts</li>
<li>twitter-color-emoji</li>
<li>urlwatch</li>
</ul>
</details>

**Review by mweinelt — APPROVED** [review]
